presto-spi/src/main/java/com/facebook/presto/spi/statistics/Reliability.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+public final class Reliability
```

Comment:
What exactly is `Reliability`? Please add some definition (formal would be best). If it doesn't have formal definition, then it doesn't have much use. If connectors can return arbitrary values, how optimizer knows what is reliable and what isn't? How it can compare reliabilities from different sources?